### ISSUE TITLE
Sound Device Options

### ISSUE THREAD

**Issue by Srdjan-V** [Krutoy242/Enigmatica2Expert-Extended#31]
### A mod that allows me to change my sound output device after opening minecraft would be great.

I have personally used Sound Device Options in E2E. If i remember correctly it had a bug with listing the outputs twice, but it did work.
https://www.curseforge.com/minecraft/mc-mods/more-sound-config

I have also found this one but i never used it.
https://www.curseforge.com/minecraft/mc-mods/sound-reloader

### This is also a mod i found while googling and i have no idea if its any good, since it's not on curseForge.
https://sk1er.club/mods/patcher
And its github page:
https://github.com/Sk1erLLC/Patcher

**Comment by Krutoy242** [comment]
`E2E-E` already have sound device reloading feature!

Press `Shift+F8` ingame.

**Comment by Krutoy242** [comment]
> Is that in the loading screen tips? 

Yes, it is, there https://github.com/Krutoy242/Enigmatica2Expert-Extended/blob/master/resources/enigmatica/lang/en_us.lang#L263-L331

**Comment by Srdjan-V** [comment]
Thanks, and have a nice day.
